pkg/apis/topology/v1alpha2/types.go

```diff
@@ -0,0 +1,159 @@
+package v1alpha2
+
+import (
+	_ "github.com/gogo/protobuf/gogoproto"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type TopologyManagerPolicy string
```

Comment:
you can remove that

Reply:
thought about that, but I prefer a very soft deprecation path (I think there are consumers of this type). I'd rather remove on `v1beta1` and deprecate till that point